src/utils/defaultSettings.js

```diff
@@ -71,6 +71,7 @@ if (!accelerators['miniplayer-open-close']) {
 settingsProvider.setInitialValue('has-updated', false)
 
 settingsProvider.setInitialValue('discord-presence-settings', {
+    default: true,
```

Comment:
This needs to be modified to check if there is already a `discord-presences-settings.default` as the `setInitialValue` will not add it for existing YTM Desktop Users.

*Additionally I recommend setting this to an "alternativeLayout" style name instead of Default as the Default should be default, and people should optionally opt into a different style and know what they're opting into.*